calculator.js

```diff
@@ -97,30 +99,25 @@ periodButton.addEventListener("click", function() {
 negativeValueButton.addEventListener("click", function() {
     const currentValueOfResult = result.textContent;
     if (currentValueOfResult !== "" && currentValueOfResult[0] !== "-") { result.textContent = "-" + currentValueOfResult };
-    if (currentValueOfResult !== "" && currentValueOfResult[0] === "-") { result.textContent = currentValueOfResult.slice(1) };
-
+    if (currentValueOfResult !== "" && currentValueOfResult[0] === "-") { result.textContent = currentValueOfResult.slice(0) };
```

Comment:
Why the slice value changes to 0?

Reply:
It's not its first time 
maybe there was pulling problems !!

Reply:
Can you check what the value of this on your local master branch ?

Reply:
I knew why It became 0 and the reason behind it was that when i was facing branch and git hub errors I copied all the codes in a new notepad file 
then when i fixed the branch problems 
I copied and pasted the codes from notepad again